pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -908,6 +920,10 @@ class MultiControlledX(Operation):
 
     >>> qml.MultiControlledX(control_wires=[0, 1, 2, 3], wires=4, control_values='1110')
```

Comment:
I think we can actually just delete this whole section.  `MultiControlledX` used to inherit from `ControlledQubitUnitary`, but it doesn't anymore, so that part is outdated.

We also don't need to have an example showing the outdated way of using `control_wires`. In the docstring, we should only demonstrate the way we want people to be creating `MultiControlledX`.